doc/can-observe.md

```diff
@@ -309,6 +309,54 @@ class WidgetViewModel {
 }
 ```
 
+## Extending can-observe with Observables
```

Comment:
We need a better term than `Observables` ... something like `with rich property behavior`.

Reply:
also it couldn't hurt to explain what the goal is here before jumping into technical details.